docs/admin/engines/private-engines.rst

```diff
@@ -55,11 +53,5 @@ and come up with new solutions which fit your use case.
 Acknowledgement
 ===============
 
-This development was sponsored by `Search and Discovery Fund`_ of `NLnet Foundation`_ .
```

Comment:
"Next steps" section can be removed.

Reply:
Thanks .. done .. I will push  it later .. there is a lot of other doc-stuff I am working on ATM ..